macros_doc.md

```diff
@@ -9,15 +34,40 @@ such as Excel.
 
 MkDocs (the tool we use to turn the markdown version of the specification into
 HTML pages) supports macros. In the BIDS specification document, we use these
-macros to standarize the format of items such as tables and examples.
+macros to standardize the format of items such as tables and examples.
```

Comment:
I would move the example of writing the file tree to here.

The following is an example of a macro used to create consistent "file tree" layouts in the documentation.
The macro takes a single parameter, the directory tree to be displayed in JSON format.
If you insert the following in the BIDS markdown document:

```
{{ MACROS___make_filetree_example(

   {
   "sub-01": {
      "func": {
         "sub-control01_task-nback_bold.json": "",
         },
      }
   }

) }}
```
The result would be rendered in the specification document as:

```Text
└─ sub-01/
   └─ func/
      └─ sub-control01_task-nback_bold.json
```

Reply:
For the moment, I am not moving it but copy pasting it. I feel that because this is an FAQ, it is very possible people will not read from top to bottom.